styles/sass/sf-additional.scss

```diff
@@ -2355,3 +2380,115 @@ sd-multi-image-edit {
         border-radius: 0 !important;
     }
 }
+.sd-flex--align-self-stretch {
+    align-self: stretch !important;
+}
+.subnav {
+    [sd-searchbar] {
+        display: contents;
+    }
+}
+
+.publish-queue {
+    .flat-searchbar.extended {
+        max-width: 380px;
+    }
+}
+
+
+// Logical Properties & Values for margins and paddings
+// Margin
+.sd-margin-s {
+    &--0-5 {
+        margin-inline-start: $sd-base-increment / 2;
+    }
+    &--1 {
+        margin-inline-start: $sd-base-increment * 1;
+    }
+    &--1-5 {
+        margin-inline-start: $sd-base-increment * 1.5;
+    }
+    &--2 {
+        margin-inline-start: $sd-base-increment * 2;
+    }
+    &--3 {
+        margin-inline-start: $sd-base-increment * 3;
+    }
+    &--4 {
+        margin-inline-start: $sd-base-increment * 4;
+    }
+    &--5 {
+        margin-inline-start: $sd-base-increment * 5;
+    }
+}
+.sd-margin-e {
+    &--0-5 {
+        margin-inline-end: $sd-base-increment / 2;
+    }
+    &--1 {
+        margin-inline-end: $sd-base-increment * 1;
+    }
+    &--1-5 {
+        margin-inline-end: $sd-base-increment * 1.5;
+    }
+    &--2 {
+        margin-inline-end: $sd-base-increment * 2;
+    }
+    &--3 {
+        margin-inline-end: $sd-base-increment * 3;
+    }
+    &--4 {
+        margin-inline-end: $sd-base-increment * 4;
+    }
+    &--5 {
+        margin-inline-end: $sd-base-increment * 5;
+    }
+}
+
+// Padding
+.sd-padding-s {
```

Comment:
it'd be clearer to use `padding-start` instead of `padding-s`.  padding-s might as well be understood as "padding small". I'd use the full word for `e` too.

Reply:
Done